guides/11_Priority_Based_Downlink_Policy.md

```diff
@@ -113,6 +114,29 @@ The following presets are provided:
 * *Unstable Network Preset*: `VideoPriorityBasedPolicyConfig.UnstableNetworkPreset` configures quick network issue response delay and slow recovery delay. If expecting a lot of mobile usage, builders can choose unstable network preset. It has less confidence in the network so it takes longer wait when network is recovered in case the network is inconsistent.
 * *Stable Network Preset*: `VideoPriorityBasedPolicyConfig.StableNetworkPreset` configures slow network issue response delay and quick recovery delay. Select stable network preset when the network is expected to stay stable, for example in unified communications client or Ethernet connections. It will try to 'wait-out' network downturns in case they are a temporary spike.
 
+### Enabling Server Side Network Adaption Features <span id="server-side-network-adaption"><span>
+
+`VideoPriorityBasedPolicy` has support for configuring additional features on the backend. These features are currently opt-in but may become defaults in the future. To enable pacing and probing on the existing receive side estimator, and disable the policy based stream probing which can often oversubscribe over network constraints and lead to video freezes, set the following on `VideoPriorityBasedPolicyConfig` before injecting:
```

Comment:
I added a note about the fact the value may be override if set to default, but I didn't not bother mentioning the downlink policy functions since those are largely internal. They work fine if not set and if a builder really wants to fiddle around there are docstrings :).